src/module/actor/base.ts

```diff
@@ -1272,7 +1275,6 @@ interface ActorPF2e extends Actor<TokenDocumentPF2e, ItemTypeMap> {
         data: PreCreate<foundry.data.ActiveEffectSource>[] | PreCreate<ItemSourcePF2e>[],
         context?: DocumentModificationContext
     ): Promise<ActiveEffectPF2e[] | ItemPF2e[]>;
-
```

Comment:
This one too

Reply:
Guess I got overzealous with removing the inserted lines. Pushed update.